.github/workflows/ci.yaml

```diff
@@ -349,22 +349,6 @@ jobs:
             ./target/${{ matrix.target }}/release/clarinet test --manifest-path ${testdir}/Clarinet.toml
           }
 
-      - name: Upload Cargo Artifact to GH release (Linux)
-        uses: svenstaro/upload-release-action@v2
-        if: startsWith(github.ref, 'refs/heads/main') && matrix.os != 'windows-latest'
-        with:
-          repo_token: ${{ secrets.GITHUB_TOKEN }}
-          file: clarinet-${{ env.SHORT_TARGET_NAME }}.tar.gz
-          tag: ${{ github.ref }}
-
-      - name: Upload Cargo Artifact to GH release (Windows)
-        uses: svenstaro/upload-release-action@v2
-        if: startsWith(github.ref, 'refs/heads/main') && matrix.os == 'windows-latest'
-        with:
-          repo_token: ${{ secrets.GITHUB_TOKEN }}
-          file: clarinet-${{ env.SHORT_TARGET_NAME }}.msi
-          tag: ${{ github.ref }}
-
```

Comment:
Pretty sure these `svenstaro/upload-release-action` actions aren't needed anymore, since artifacts are now being uploaded to the release [here](https://github.com/hirosystems/clarinet/pull/495/files#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133ddR718).